cabal-install/src/Distribution/Client/Main.hs

```diff
@@ -296,6 +312,12 @@ warnIfAssertionsAreEnabled =
     assertionsEnabledMsg =
       "Warning: this is a debug build of cabal-install with assertions enabled."
 
+-- | Core worker, similar to simpleMainHelper in
+-- Cabal/Distribution.Simple
+--
+-- With an exception-handler (topHandler),
```

Comment:
Curious, is there a reason why you keep your comment lines way shorter than the 80max customary ?limit?

Reply:
I'm attempting to improve readability by verticalizing comments, within reason. The goal would be for someone viewing the comments manually to not have to scan too far for the entire semantic unit, with the newlines providing an obvious space to pause.

A further issue is, say, disambiguation. Cabal/Distribution.Simple naturally would want a period at the end, but is the period attached to the module name or the sentence?

I have no issue with editing, if this is seen to be an issue.